docs/en/pid-issuing.rst

```diff
@@ -0,0 +1,511 @@
+    :target: http://www.plantuml.com/plantuml/png/hLHHQoCt47xNhnXo1xt0Xhu7V5noSI-cePWqnxqK5dbtx1sobhRIA3xtrtSaUHUjcnuf5FpWrSmttyptQVJ9itOSzhtIWQq9-mqwfGOveuO6RHWkldfq5w0z1AKszdi--Cc3tqlMq5c35_3kWNOTt_O1vzXRPmyJNCzZttJV8yFAI0kcmPHpK6XQ41RrQkxQsHTgJ_KsPpdhrTqyWSuIS7LtypkKQV27UNB1CxRmG9uJYfLIYqLA8-y3cHt0Og7K0Qu-GhW6ks5D1eusk9Ya0bn9u70DH_HAEMCPmRv8xKF5yExR_UffgKuO3RsmWDt2o5es4qTFSE6w-YXCZJLRSdlW3i5ZZmsJDOAUerGGVg6tpsVi9Awiljm_VNbSBdzJPsf3qs7pxAMsvaWmbuKyHEM4AyL-GJdKYLsRTbI0sCCUMRUQTTaxH3b0sdUeMGh_-dafexG8qXPj2TqfXoHdVVC8Z_XdGC_mmQ4VhF5Oyt702fgUq71DRPNWTTF54SmEpxzhwQApakPofGcglKSNsougbiUBzrcLKPGRPe6dGTzVGgacLPkPcZErpgNCHT-Yve_hUDePpS6X8Cl2HPiG3KbElEYUMk8Z55OeodTp4ZhhAqfdeUhWg0ByFf18NvEv5CWXGOA-XfiWgZhwAnDDBXMs56KkLT09IVyPgEP2Xt5fWfTlBRK4HKqkTY3k8E_nFUswD7qJPp9Sp4sd6U2ibqA1U8CgY2hYrFhXZVd57T-XYLuYtAtj6fxn-A-nmLbnSMlTVDFOFgC1QNsm93CKg_4aaKczF7XHg5HpvxJXyOhRTjoFSTt-btM8dhNai1bDepncxhDKIZSDUb-dmQeyN-OhBffmryOAnVIdnYdughvRYNLHu2HX8ysWOTBzFz-Rh8_y1Yk3UHzz_91PidL5_e_TANacYmg7Fa_HLsmgPOZNT6xUXAnRt9AXg74L-VUQgvnrMQx5-9RVFdnTgaVSaMSv5RU0EopQl7BOQyOM59bSJGgdkV9jo6lqiyCNs-GOb7kJfIadAumzTLUmlTd6AEhqG8aOQP62pozUaUVPEYpT_IJDXNt_Dm00
+
+    PID Issuing - Detailed flow
+
+**Steps 1-4:** The User has selected a PID Provider, and the Wallet Instance obtains the metadata for the selected PID Provider.
+
+.. note::
+
+    **Federation Check:** The Wallet Instance needs to check if the PID Provider is part of Federation and then it can consume its Metadata.
+
+**Steps 5-6:** The Wallet Instance creates a fresh PKCE code verifier and sends this along in a *pushed authorization request* using the OIDC request parameter (see :rfc:`9126` Section 3) to the pushed authorization endpoint of the PID Provider. The Wallet Instance signs this request using its attested private key. We assume that a standard OAuth2 client authentication method should be involved as the pushed authorization endpoint is a protected endpoint. The client authentication can be based on the model defined in [:rfc:`7521`] using the Wallet Instance Attestation JWS inside the client_assertion parameter. The authorization_details [RAR :rfc:`9396`] parameter is extended to allow Wallet Instance to specify types of the credentials when requesting authorization for PID issuance.
+
+The following is a non-normative example of the PAR.
+
+.. code-block::
+
+    POST /as/par HTTP/1.1
+    Host: pid.it
+    Content-Type: application/x-www-form-urlencoded
+    
+    response_type=code
+    &client_id=$thumprint-of-the-jwk-in-the-cnf-wallet-attestation$
+    &code_challenge=E9Melhoa2OwvFrEMTJguCHaoeK1t8URWbuGJSstw-cM
+    &code_challenge_method=S256
+    &request=eyJhbGciOiJSUzI1NiIsImtpZCI6ImsyYmRjIn0.ew0KIC Jpc3MiOiAiczZCaGRSa3F0MyIsDQogImF1ZCI6ICJodHRwczovL3NlcnZlci5leGFtcGxlLmNvbSIsDQo gInJlc3BvbnNlX3R5cGUiOiAiY29kZSBpZF90b2tlbiIsDQogImNsaWVudF9pZCI6ICJzNkJoZFJrcXQz IiwNCiAicmVkaXJlY3RfdXJpIjogImh0dHBzOi8vY2xpZW50LmV4YW1...
+    &client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
```

Comment:
We used for the wallet instance attestation:
`urn:ietf:params:oauth:grant-type:wallet-instance-attestation` 
do we agree on a unique format? @peppelinux

Reply:
indeed

@fmarino-ipzs please chage to `urn:ietf:params:oauth:grant-type:wallet-instance-attestation` even because the wallet instance attestation is not a bearer token :-)

Reply:
I was wondering if it is better to consider `urn:ietf:params:oauth:client-assertion-type:jwt-key-attestation` according to[ this spec](https://vcstuff.github.io/draft-looker-oauth-attested-key-based-client-authentication/draft-looker-oauth-attested-key-based-client-authentication.html#name-sub-namespace-registration-), instead of `urn:ietf:params:oauth:grant-type:wallet-instance-attestation`.  
What do you think?

Reply:
As just discussed with @peppelinux we decide to adopt `urn:ietf:params:oauth:client-assertion-type:jwt-key-attestation` in support of vcstuff. @grausof please align this in the wallet instance solution.